File: app/src/main/java/ca/ivasenko/mobiletest/resources/model/Address.java
package ca.ivasenko.mobiletest.resources.model;

public class Address
{
    private String address1;
    private String label;
    private String zipCode;
    private String city;
    private String state;
    private String country;
    private Gps gps;

    public Address()
    {
    }

    public String getAddress1()
    {
        return address1;
    }

    public void setAddress1(String address1)
    {
        this.address1 = address1;
    }

    public String getLabel()
    {
        return label;
    }

    public void setLabel(String label)
    {
        this.label = label;
    }

    public String getZipCode()
    {
        return zipCode;
    }

    public void setZipCode(String zipCode)
    {
        this.zipCode = zipCode;
    }

    public String getCity()
    {
        return city;
    }

    public void setCity(String city)
    {
        this.city = city;
    }

    public String getState()
    {
        return state;
    }

    public void setState(String state)
    {
        this.state = state;
    }

    public String getCountry()
    {
        return country;
    }

    public void setCountry(String country)
    {
        this.country = country;
    }

    public Gps getGps()
    {
        return gps;
    }

    public void setGps(Gps gps)
    {
        this.gps = gps;
    }
}

File: app/src/main/java/com/rodionov/igor/mobiletest/model/Address.kt
package com.rodionov.igor.mobiletest.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "addresses")
class Address {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    var id: Int = 0
    @ColumnInfo(name = "address1")
    var address1: String? = null
    @ColumnInfo(name = "label")
    var label: String? = null
    @ColumnInfo(name = "zipCode")
    var zipCode: String? = null
    @ColumnInfo(name = "city")
    var city: String? = null
    @ColumnInfo(name = "state")
    var state: String? = null
    @ColumnInfo(name = "country")
    var country: String? = null
    var gps: GpsEntity? = null
}
